container/BUILD

```diff
@@ -26,6 +26,71 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
+py_binary(
+    name = "extract_config",
+    srcs = ["extract_config.py"],
+    legacy_create_init = False,
+    python_version = "PY2",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@containerregistry",
+    ],
+)
+
+py_binary(
```

Comment:
Why not depend on https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/hash/BUILD#L16 instead of needing our own here?